readme.md

```diff
@@ -131,12 +137,18 @@ You can fill these bowls (or any group:food_bowl) with vegetables to
 craft an uncooked vegetable stew:
 
 empty       empty     empty
-grean_beans potato    tomato
+green_beans potato    tomato
+empty       clay_bowl empty
+
+or an uncooked ratatouille:
+
+empty       empty     empty
+pumpkin     eggfruit  tomato
 empty       clay_bowl empty
 
-The uncooked vegetable stew obviously needs to be cooked as well in
-an oven. The resulting Vegetable Stew bowl gives a lot of hears back,
-which is worth the effort.
+The uncooked meals obviously needs to be cooked as well in an oven. The
+resulting vegetable dishes give a lot of hearts back, which is worth the
+effort.
 
 The watering can can be made as follows:
 
```

Comment:
Can you post some screenshots of how this looks in-game? The PNG's, obviously. Did you draw the images yourself?